english/batman/batman.md

```diff
@@ -23,7 +23,7 @@ this realization, then you go:
 
 Bruce Wayne: It's not who I am underneath, but what I do that defines me.
 
-Commissioner Gordon: He's the hero Gotham deserves but not the one that it needs right now.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.